tests/vendor_tests/redhat/rhbz1928628.py

```diff
@@ -7,7 +7,8 @@
 # See the LICENSE file in the source distribution for further information.
 
 
-from report_tests.plugin_tests.networking import NetworkingPluginTest
+from tests.report_tests.plugin_tests.networking.networking import \
+    NetworkingPluginTest
```

Comment:
When the tests are run, we already have `PYTHONPATH=tests`, so curious why we need this change?

Reply:
This is to fix `E0401: Unable to import 'report_tests.plugin_tests.networking.networking' (import-error)`. If I set `PYTHONPATH=tests`, it still complains because of missing module `networking` (i.e. `tests` can be dropped but the extra `networking` is still needed).

Since this isn't currently raised in github CI, I guess I can drop this change altogether. But still wonder if this test is run at all as I can't see how it'd work without `networking` at the end.

Reply:
cool, sounds good to me